vta/tutorials/matrix_multiply.py

```diff
@@ -52,7 +52,7 @@
 
 # We configure both the bitstream and the runtime system on the Pynq
 # to match the VTA configuration specified by the vta_config.json file.
-if env.TARGET == "pynq":
+if env.TARGET == "pynq" or env.TARGET == "de10nano":
```

Comment:
To keep the code maintainable as we scale the number of supported FPGAs, we should a function in the environment.py library that essentially checks the target against a list of FPGA target and used as such: `env.target_is_fpga()` for all of the FPGA delineations, and `env.target_is_sim()` for tsim, fsim

Reply:
Totally agree.
Do you want me to remove the de10nano check at this point.
I just wanted to document how to invoke a test with the de10nano target.

Reply:
Ok, that is fair, thanks